public/sass/pages/_explore.scss

```diff
@@ -4,26 +4,6 @@
   color: $orange-dark !important;
 }
 
-// TODO: check if this is used
-.explore {
-  display: flex;
-  flex: 1 1 auto;
-  flex-direction: column;
-}
-
-.explore + .explore {
-  border-left: 1px dotted $table-border;
-}
```

Comment:
removing this removed the vertical border between explore panes in split mode:
<img width="1215" alt="Screenshot 2022-07-12 at 18 33 29" src="https://user-images.githubusercontent.com/1170767/178545691-7eecca9c-309c-44a8-a92b-9f471736e76e.png">

while before we had a light border between the panes:
<img width="882" alt="Screenshot 2022-07-12 at 18 34 28" src="https://user-images.githubusercontent.com/1170767/178545866-7eee17ff-1a12-454a-bfd0-048a0242b8a4.png">

Reply:
Oh, there was the line. I restored it for the split view now.